src/mapproxyUtils/mapproxySeed.ts

```diff
@@ -0,0 +1,29 @@
+import { Logger } from '@map-colonies/js-logger';
+import { inject, singleton } from 'tsyringe';
+import { SERVICES } from '../common/constants';
+import { IConfig, ISeedOptions } from '../common/interfaces';
+
+@singleton()
+export class MapproxySeed {
+  private readonly mapproxyYamlDir: string;
+  private readonly seedYamlDir: string;
+
+  public constructor(@inject(SERVICES.LOGGER) private readonly logger: Logger, @inject(SERVICES.CONFIG) private readonly config: IConfig) {
+    this.mapproxyYamlDir = this.config.get<string>('mapproxy.mapproxyYamlDir');
+    this.seedYamlDir = this.config.get<string>('mapproxy.seedYamlDir');
+  }
+
+  //todo - not implemented on skeleton phase - will developed on phase 2
+  public async runSeed(task: ISeedOptions): Promise<void> {
```

Comment:
add try catch for better error handling and to document the failed seed task params

Reply:
done